src/cook-web/src/components/shifu-setting/ShifuSetting.tsx

```
@@ -90,7 +90,13 @@ interface Shifu {
 }
 
 const MIN_SHIFU_PRICE = 0.5;
-
+const TEMPERATURE_MIN = 0;
+const TEMPERATURE_MAX = 2;
+const TEMPERATURE_STEP = 0.1;
+const FLOAT_EPSILON = 1e-6;
+const TEMPERATURE_MIN = 0;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'TEMPERATURE_MIN'. Consider to delete it or rename it.

'TEMPERATURE_MIN' is defined here:


(lint/suspicious/noRedeclare)